src/steamship/invocable/mixins/blockifier_mixin.py

```diff
@@ -34,11 +33,11 @@ def blockify(
         """
 
         file = File.get(self.client, _id=file_id)
-        update_file_status(self.client, file, "Blockifying")
+        file.add_or_update_metadata("status", "Blockifying")
```

Comment:
Maybe `add_or_update_tag`? (If I'm right above that the `metadata` object has different semantics than the `tags` array

Reply:
Correct. Yes. I do think it's a nice addition to have. But since the indexer pipeline is the only one using it we can stick I'll change the update_file_status to be a bit more flexible.